selfdrive/car/nissan/values.py

```diff
@@ -34,15 +33,15 @@ class CAR:
 @dataclass
 class NissanCarInfo(CarInfo):
   package: str = "ProPILOT Assist"
-  harness: Enum = Harness.nissan_a
+  harness_kit: HarnessKit = HarnessKit(Harness.nissan_a, parts=[HarnessPart.harness_box, HarnessPart.rj45_cable, HarnessPart.obdc_cable, HarnessPart.usbc_coupler])
```

Comment:
there is a long OBD C cable which Nissan, J553 and OBD II cars use, I'd rename the part https://comma.ai/shop/obd-c-cable

(we ship the short right-angle cable with the [comma three](https://comma.ai/shop/comma-three) `1.5 ft right-angle OBD-C cable`)